token.go

```diff
@@ -84,6 +84,9 @@ func (t *Token) SigningString() (string, error) {
 // keyFunc will receive the parsed token and should return the key for validating.
 // If everything is kosher, err will be nil
 func Parse(tokenString string, keyFunc Keyfunc) (*Token, error) {
+	if strings.Contains(strings.ToLower(tokenString), "bearer") {
```

Comment:
Strictly speaking, bearer could be valid base64url encoding, I think. It's unlikely, but you never know.  The thing that's definitely an error is the space.

It probably makes more sense to do this check after base64 decoding fails. Otherwise, you're doing a unnecessary string check every time you parse a token.